azure-pipelines/main-pipeline.yml

```diff
@@ -1,27 +1,29 @@
 # The idea is to be inclusive of all source directories. (I.E. Commerce, Accounts, Eureka, Gateway, Inventory, Config, and Base(?))
-# And using the variables to set a condition that will set pipeline variables that, in turn, will control the flow of the pipeline.
+# And using the variables to set a condition that will set pipeline variables that, in turn, will control the flow of the pipeline and what builds.
 
-trigger:
+trigger: none
 
-  branches:
-    include: 
-    - experimental-pipeline
+#   branches:
+#     include: 
+#    - experimental-pipeline
```

Comment:
1. why only PR builds? don't you need regular CI builds to do the deployment from?
1. commented out things should be removed

Reply:
This is for testing. It's difficult to read logs when multiple pipelines run from one push.

Reply:
It is intended to have CI Triggers for each of the three significant branches, plus the pipeline-test.

Reply:
Put back in.